package.json

```diff
@@ -143,8 +143,9 @@
     "ddos": "^0.2.1",
     "decentraland-commons": "^5.1.0",
     "decentraland-crypto-middleware": "^1.0.2",
-    "decentraland-dapps": "^16.7.0",
+    "decentraland-dapps": "^16.8.0",
     "decentraland-server": "^3.0.0",
+    "decentraland-ui": "^4.11.3",
```

Comment:
No need to install `decentraland-ui` here, it's already inside `decentraland-dapps`